packages/react-router/modules/__tests__/Route-test.js

```diff
@@ -34,6 +34,23 @@ describe('A <Route>', () => {
 
     expect(node.innerHTML).toNotContain(TEXT)
   })
+
+  it('supports preact by nulling out children prop when empty array is passed', () => {
+    const TEXT = 'Mrs. Kato'
+    const node = document.createElement('div')
+
+    ReactDOM.render((
+      <MemoryRouter initialEntries={[ '/' ]}>
+        <Route path="/" render={() => (
+          <h1>{TEXT}</h1>
+        )}>
+          {[]}
```

Comment:
Oh wait, duh. Of course the warning isn't firing because you `null` out children before we get that far. Ignore me :)

Question: does Preact *always* give you a `children` prop, @developit? i.e. if I render a component w/out any children like `<MyComponent/>` does Preact give me an empty `children` array?